workbench/src/main/setupAddRemovePlugin.js

```diff
@@ -178,7 +177,7 @@ export function setupAddPlugin(i18n) {
         );
         logger.info('successfully added plugin');
       } catch (error) {
-        console.log(error);
+        logger.info(error);
```

Comment:
For some reason, the `logger.info` gave more useful information when I was debugging this.